compiler_gym/random_search.py

```diff
@@ -164,7 +162,7 @@ def random_search(
 
     print(
         f"Started {len(workers)} worker threads for "
-        f"{benchmark_name} ({humanize.intcomma(metadata['num_instructions'])} instructions) "
+        f"{benchmark_name} ({humanize.intcomma(metadata['reward'])}) "
         f"using reward {reward_space_name}."
```

Comment:
Nitpick: this prints out the rewards space name twice:

```
$ bazel run -c opt //compiler_gym/bin:random_search -- --env=llvm-ic-v0 --benchmark=cBench-v1/dijkstra
Started 24 worker threads for benchmark://cBench-v1/dijkstra (IrInstructionCountOz) using reward IrInstructionCountOz.
```

perhaps just:

```suggestion
        f"{benchmark_name} using reward {reward_space_name}."
```